docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+
+# Protoc binary
+FROM node:lts-stretch as protoc-bin
+ARG PROTOC_VERSION
+RUN mkdir /installer
+RUN wget https://github.com/protocolbuffers/protobuf/releases/download/v$PROTOC_VERSION/protoc-$PROTOC_VERSION-linux-x86_64.zip -O protoc.zip
+RUN mkdir -p /installer/protoc
+RUN unzip -o protoc.zip -d /installer/protoc/
+
+
+# Node
+FROM node:lts-stretch as node
+
+COPY package.json .
+COPY yarn.lock .
+RUN yarn install
+
+
+# Java
+FROM gradle:latest as java
+COPY build.gradle .
+RUN gradle download
+RUN mv installer /
+
+
+# Go
+FROM golang:1.15 as gopher
+ARG GO_GRPC_TOOLS_VERSION
+
+RUN mkdir /temp
+RUN cd /temp
```

Comment:
does nothing because each RUN is done in its own shell
just use WORKDIR

Reply:
You start in the default `WORKDIR` which corresponds to hte `$GOPATH`, the default place where go dependencies are stored. Go throws an error when it sees an `go.mod` file in there, thats why this steps are required. @moritzzimmer  Do you have an opinion or a suggestion in that case?